adam_core/orbits/orbits.py

```diff
@@ -169,7 +167,7 @@ def from_df(
         cls : `~adam_core.orbits.Orbits`
             Orbits class.
         """
-        data = cls._dict_from_df(
+        data = CoordinateMembers._dict_from_df(
```

Comment:
Same here, is there something with the inheritance that I can't remember?

Reply:
Fixed.